ocpp/messages.py

```diff
@@ -32,7 +33,7 @@ def unpack(msg):
     Unpacks a message into either a Call, CallError or CallResult.
     """
     try:
-        msg = json.loads(msg)
+        msg = json.loads(msg, parse_float=decimal.Decimal)
```

Comment:
Is it ok to always parse floats here but only schemas in a few instances?

Reply:
Good catch, I need to verify that.

Reply:
I started working on this a bit yesterday, and I got stuck on that part of porting the code over. If it's an issue I have an idea.

Reply:
I've removed the argument `parse_float=decimal.Decimal`. It is not needed in this place.

If a message requires special validation regarding the floats the message is reparsed again using `parse_float=decimal.Decimal`. See line 138 of this file.